lib/model.js

```diff
@@ -290,7 +291,7 @@ module.exports = (function() {
     this.Instance.prototype.validators = {};
 
     Utils._.each(this.rawAttributes, function(definition, name) {
-      var type = definition.type._typeName || definition.type;
+      if (typeof definition.type === "function") definition.type = new definition.type();
```

Comment:
Hmm, should this be done several times? Is it not better to convert all type defs to instances at one place?